docs/modules/plugins/pages/plugin-web-app.adoc

```diff
@@ -299,6 +299,221 @@ When I scan barcode from context and save result to scenario variable `qrCodeLin
+
+=== Hover mouse over the element
+
+Moves a mouse cursor on the element.
+
+[source,gherkin]
+----
+When I hover mouse over element located by `$locator`
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+When I hover mouse over element located `$locator`
+----
+
+* `$locator` - The <<_locator,locator>> to an element to hover.
+
+.Step example
+[source,gherkin]
+----
+When I hover mouse over element located by `id(hoverme)`
+----
+
+=== Check each element has a specific number of children elements
+
+Checks that each element specified by the locator contains an exact number of visible child elements specified by another locator
+
+[source,gherkin]
+----
+Then each element with locator `$elementLocator` has `$number` child elements with locator `$childLocator`
+----
+
+* `$elementLocator` - The <<_locator,locator>> to the parent element.
+* `$number` - The expected number of children elements
+* `$childLocator` - The <<_locator,locator>> to the child elements.
+
+.Step example
+[source,gherkin]
+----
+Then each element with locator `id(menu)` has `5` child elements with locator `id(menu-item)`
+----
+
+=== Check an element CSS property
+
+Checks the context element has the expected CSS property.
+
+The context can be set by the <<_change_context,corresponding steps>>.
+
+[source,gherkin]
+----
+Then context element has CSS property `$cssName` with value that $comparisonRule `$cssValue`
+----
+
+* `$cssName` - A name of the CSS property.
+* `$comparisonRule` - CSS property xref:parameters:string-comparison-rule.adoc[comparison rule].
+* `$cssValue` - The expected value of the CSS property.
+
+.Change context to element and verify it color CSS property value
+[source,gherkin]
+----
+When I change context to element located by `id(rainbow)`
+Then context element has CSS property `color` with value that is equal to `rgba(0, 0, 0, 1)`
+----
+
+=== Compare sizes of the elements
+
+Check that all elements found on the page by the given locator have the same dimension: width / height.
+
+NOTE: Element's dimension is measured in *pixels* and means the size that given element occupies on the web-page
+in a browser. Element size may vary in different browsers, it also depends on screen resolution, page scaling,
+scripts running on the page.
+
+[source,gherkin]
+----
+Then each element located by `$locator` has same `$dimension`
+----
+
+_Deprecated syntax (will be removed in VIVIDUS 0.7.0)_:
+[source,gherkin]
+----
+Then each element located `$locator` has same `$dimension`
+----
+
+* `$locator` - The <<_locator,locator>> to find an elements to compare.
+* `$dimension` - The elements dimension to compare. Possible values: _width_, _height_
+
+.Check that all menu items have the same size (width and height)
+[source,gherkin]
+----
+Then each element located `id(context-menu-item)` has same `width`
+Then each element located `id(context-menu-item)` has same `height`
+----
+
+=== Check element width in a percentage
```

Comment:
in this case the width is calculated relatively to body, please add this in docs